db/seeds.rb

```diff
@@ -1,7 +1,171 @@
+          "Quality Assurance",
+          "Quality Control",
+          "Quantitative Research",
+          "Quantitative Reports",
+          "Refinishing",
+          "Remodeling",
+          "Repairing",
+          "Reporting",
+          "Restoration",
+          "Security",
+          "Servers",
+          "Storage",
+          "Software",
+          "Solution Delivery",
+          "Sorting",
+          "Statistical Analysis",
+          "Structures",
+          "Systems Analysis",
+          "Technical Support",
+          "Technical Writing",
+          "Technology",
+          "Testing",
+          "Tools",
+          "Training",
+          "Troubleshooting",
+          "Usability"]
+
+
+INTERESTS = ["Sports such as football, baseball, golf, soccer, tennis etc.",
+            "Photography/Videography",
+            "Movies and Theater",
+            "Video games",
+            "Traveling and Exploring different places",
+            "Reading",
+            "Listening to music",
+            "Playing a musical instrument",
+            "Dancing",
+            "Acting",
+            "Surfing the Internet/Blogging/Programming",
+            "Gardening",
+            "Collecting",
+            "Bowling",
+            "Iceskating",
+            "Taking care of pets",
+            "Trekking",
+            "Fitness",
+            "Extreme sports and activities (such as sky diving, mountain climbing etc.)",
+            "Wine tasting",
+            "Social work.",
+            "Driving, go-karting. Riding motorbikes",
+            "Car restoration and repairing",
+            "Mountain climbing, biking.",
+            "Fishing.",
+            "Indoor games such as chess, cards (poker), pool or billiards.",
+            "Carpentry.",
+            "Brewing",
+            "Fantasy sports",
+            "Hunting",
+            "Building models and miniatures",
+            "Camping",
+            "Body Building/Fitness",
+            "Coaching and teaching.",
+            "Astrology",
+            "Astronomy",
+            "Social work.",
+            "Fashion designing",
+            "Hair dressing/hair styling",
+            "Nail art",
+            "Photography",
+            "Interior designing",
+            "Gardening",
+            "Yoga, meditation.",
+            "Baking/cooking.",
+            "Diet and nutrition.",
+            "Health and fitness",
+            "Knitting, sewing.",
+            "Dancing.",
+            "Singing/music.",
+            "Coaching and teaching.",
+            "Jewelry making.",
+            "Painting.",
+            "Pottery, sculpture.",
+            "Astrology, tarot card reading.",
+            "Knitting",
+            "Social work",
+            "Candle making",
+            "Iceskating"]
+
+
+# if Rails.env.development?
+#   FactoryGirl.build(:project)
+# end
+
+50.times do
+  project = FactoryGirl.create(
+  :project, name: Faker::Commerce.product_name, skill_list: SKILLS.sample(4).join(", "),
```

Comment:
Line is too long. [88/80]<br>Align the parameters of a method call if they span more than one line.<br>Indent the first parameter one step more than the previous line.